src/main/java/com/stripe/model/Mandate.java

```diff
@@ -292,4 +295,9 @@ public static Mandate retrieve(
     @SerializedName("currency")
     String currency;
   }
+
+  @Getter
+  @Setter
+  @EqualsAndHashCode(callSuper = false)
+  public static class UsBankAccount extends StripeObject {}
```

Comment:
The client_metadata for that one is wrong, it should be inside `PaymentMethodDetails` like the other ones